spec/models/item_spec.rb

```diff
@@ -15,14 +15,20 @@
 
   context "associations" do
     it { is_expected.to belong_to(:category) }
+    it { is_expected.to have_many(:order_items) }
+    it { is_expected.to have_many(:orders).through(:order_items) }
   end
 
   describe "#price_in_dollars" do
-    it "returns the price in dollars" do
-      item = create(:item, price_in_cents: 12_34)
+    let(:item) { create(:item, price_in_cents: 12_34) }
 
+    it "returns the price in dollars" do
       expect(item.price_in_dollars).to eq 12.34
     end
+
+    it "returns the price in dollars when there is more than one item" do
+      expect(item.price_in_dollars(10)).to eq 123.40
+    end
   end
 
   describe "#category_name" do
```

Comment:
We should add a new test here (and on all model specs) that a model can be saved under happy path conditions. We are testing that it needs these attributes to be valid, but we also need to make sure that a valid item is actually valid.